### PR TITLE
Update users-and-providers.md

### PR DESCRIPTION
**Pull request by andresmmujica**
Add SEAQ to Professional Services & Integrators segment

[Feature]

Changes proposed in this pull request:

- Adding new vendor to Professional Services & Integrators segment.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/main/CONTRIBUTING.md): (@andresmmujica)

This pull request needs review by: @bproffitt  @sandrobonazzola

### CODE REVIEW

File: source/community/user-stories/users-and-providers.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ The founders of [Netbulae](//www.netbulae.eu/) have experience in ICT, automatio
 [Red Hat](//www.redhat.com/en) is the corporate member of the oVirt community, responsible for opening oVirt's codebase. Besides providing support for oVirt's community, Red Hat uses oVirt as the codebase for its [Red Hat Virtualization](//www.redhat.com/en/technologies/virtualization/enterprise-virtualization) product.
 </div>
 
+<div class="case-study">
+[![SEAQ](/wp-content/uploads/2022/07/LOGO-SEAQ-cropped-transparent.png)](https://seaq.us/)
```
Comment (sandrobonazzola) [inline source/community/user-stories/users-and-providers.md:56]: This link to the logo is going to resolve to 404 error as it's a relative link

File: source/community/user-stories/users-and-providers.md [hunk 2]
```diff
@@ -52,6 +52,10 @@ The founders of [Netbulae](//www.netbulae.eu/) have experience in ICT, automatio
...
+[SEAQ](https://seaq.us/) specializes in seamless transitions to oVirt for LATAM and USA businesses, offering deep virtualization expertise to navigate away from other vendors and embrace the technology. Let us streamline your move to a robust oVirt infrastructure. [SEAQ](https://www.seaq.co/).
```
Comment (sandrobonazzola) [inline source/community/user-stories/users-and-providers.md:57]: the website doesn't mention anything related to oVirt

**Review by sandrobonazzola — CHANGES_REQUESTED** [review]

**Comment by gocallag** [comment]
I tested the change via the procedure at https://github.com/oVirt/ovirt-site/blob/2ebfb790c06d2ee172fef351536ffa78772ea0d2/CONTRIBUTING.md and it looks ok to me :-)

**Comment by sandrobonazzola** [comment]
Closing due to inactivity

**Pull request closed without merging**